app/src/main/java/com/example/playlistmaker/player/ui/activity/FragmentPlayer.kt

```diff
@@ -66,7 +86,7 @@ class PlayerActivity : AppCompatActivity() {
             viewModel.pause()
             updateButton()
         }
-        viewModel.putTime().observe(this) { timer ->
+        viewModel.putTime().observe(requireActivity()) { timer ->
```

Comment:
🍏 Можно использовать lifecycleOwner фрагмента:
```suggestion
        viewModel.putTime().observe(this.viewLifecycleOwner) { timer ->
```